src/NITTA/Model/Problems/Bind.hs

```diff
@@ -12,22 +12,44 @@ Stability   : experimental
 module NITTA.Model.Problems.Bind (
     Bind (..),
     BindProblem (..),
+    GroupBindingT (..),
 ) where
 
+import qualified Data.Set as S
 import Data.String.ToString
 import GHC.Generics
 import NITTA.Intermediate.Types
 
+data GroupBindingT = NonAlternativeBinds | AllBinds | FirstWaveBinds
```

Comment:
What is the reason for this type? If it is only for user output -- maybe string will be better here.

Anyway GroupBindingType will be better